server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
+(usually stronger) requirements. For example those security keys may require users to authenticate to that security key using PIN, or the
+security key should be attested with stronger certificate authority.
+
+Because of this, {project_name} allows administrator to configure separate `WebAuthn Passwordless Policy`. There is also separate required action
+of type `Webauthn Register Passwordless` and separate authenticator of type `WebAuthn Passwordless Authenticator`.
+
+===== Setup
+
+The setup procedure of WebAuthn passwordless support is the following :
+
+* Register new required action for WebAuthn passwordless support. Use the same steps as described <<_webauthn-register, above>>
+with the only difference, that you need to register action called `Webauthn Register Passwordless`.
+
+* Configure the policy. You can use same steps and configuration options as described <<_webauthn-policy, above>>, however you
+need to configure them in the admin console in the tab `WebAuthn Passwordless Policy`. You can configure this policy as you want, however
+typically the requirements for security key will be stronger than for two-factor policy. For example `User Verification Requirement` can
+be set to `Required` when configure passwordless policy.
+
+* Finally configure the authentication flow. Let's assume that we will use same flow called `WebAuthn Browser` as described
+<<_webauthn-authenticator-setup, above>>, but
+we will configure it like this:
+
+** The `WebAuthn Browser Forms` subflow will contain `Username Form` as first authenticator. This means that user will provide just
+his username as first step.
+
+** There will be required subflow, which can be named for example `Passwordless Or Two-factor` . This indicates that user can
+authenticate either with Passwordless WebAuthn credential or with Two-factor authentication.
 
-A user carries out the following operations on the <<_account-service, `User Account Service`>>  :
+** Flow will contain `WebAuthn Passwordless Authenticator` as first alternative.
 
-- View the `Account` page.
+** Second alternative will be subflow named for example `Password And Two-factor Webauthn`. This subflow will contain `Password Form` and
+`WebAuthn Authenticator`.
 
-===== Edit Registered WebAuthn Authenticator
+The full configuration of the flow will look like this:
 
-A user can edit the following information :
+image:images/webauthn-passwordless-flow.png[]
 
-- Label (WebAuthn authenticator's label the user entered on registering it)
+You can now add `WebAuthn Register Passwordless` as required action to some user to test this. During first authentication, user will
+be still required to use password and his second-factor WebAuthn credential. However once user registers the Passwordless WebAuthn
+credential, he will be able to choose during future authentications. If he uses his WebAuthn Passwordless credential, he won't need to
```

Comment:
If he uses his or her WebAuthn Passwordless credential, he won't need to provide the password...